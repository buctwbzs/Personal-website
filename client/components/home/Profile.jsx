import React, { Component } from 'react'
import styled from 'styled-components'

const ProfileBox = styled.section`
  display: flex;
  flex-direction: column;
  align-items: center;
  padding-top: 80px;
  width: 100%;
  height: 440px;
  >p{
    padding: 0 1em;
    text-align: center;
    line-height: 1.5em;
    color: #33344D;
    &:first-of-type{
      font-size: 2rem;
    }
    &:last-child{
      margin-top: 0;
      font-size: 1.5rem;
    }
    @media (max-width: 899px) {
      &:first-of-type{
        font-size: 1.4rem;
      }
      &:last-child{
      font-size: 1.2rem;
      }
    }
  }
`
class Profile extends Component {

  render() {
    return (
      <ProfileBox>
        <img src={`${static_url}me.gif`} alt="me" />
        <p>如果你无法简洁的表达你的想法，那只能说明你还不够了解它</p>
        <p>- 阿尔伯特·爱因斯坦</p>
      </ProfileBox>
    )
  }
}



export default Profile