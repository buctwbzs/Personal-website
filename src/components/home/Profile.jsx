import React, { Component } from 'react'
import styled from 'styled-components'
import me from '../../public/image/me.gif'


const ProfileBox = styled.section`
  display: flex;
  flex-direction: column;
  align-items: center;
  padding-top: 80px;
  height: 440px;
  >p{
    padding: 0 1em;
    text-align: center;
    line-height: 1.5em;
    color: #33344D;
    &:first-of-type{
      font-size: 28px;
      font-family: PingFangSC-Light;
    }
    &:last-child{
      margin-top: 0;
      font-size: 20px;
      font-family: PingFangSC-Thin;
    }
    @media (max-width: 899px) {
      &:first-of-type{
        margin-bottom: .5em;
        font-size: 18px;
      }
    }
    &:last-child{
      font-size: 16px;
    }
  }
`
class Profile extends Component {

  render() {
    return (
      <ProfileBox>
        <img src={me} alt="me" />
        <p>如果你无法简洁的表达你的想法，那只能说明你还不够了解它</p>
        <p>- 阿尔伯特·爱因斯坦</p>
      </ProfileBox>
    )
  }
}



export default Profile