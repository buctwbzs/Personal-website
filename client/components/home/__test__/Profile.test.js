import React from 'react'
import renderer from 'react-test-renderer'
import Profile from '../Concat'

it('Profile renders correctly', () => {
  const tree = renderer.create(<Profile />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})