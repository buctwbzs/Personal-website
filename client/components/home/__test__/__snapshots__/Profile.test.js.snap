// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile renders correctly 1`] = `
<section
  className="Concat__ConcatBox-s1clk3f5-0 bKczOW"
>
  <div>
    <a
      href="https://twitter.com/buctwbzs"
    />
    <a
      href="https://github.com/buctwbzs"
    />
    <a
      href="https://www.instagram.com/buctwbzs/"
    />
    <a
      href="mailto:buctwbzs@gmail.com"
    />
  </div>
</section>
`;
